backend/src/common/database/database.module.ts

```diff
@@ -15,7 +15,7 @@ import { WinstonLoggerService } from '../logger/winstonLogger.service';
     imports: [
         TypeOrmModule.forRootAsync({
             inject: [ConfigService, WinstonLoggerService],
```

Comment:
```suggestion
            inject: [ConfigService],
```
이거 제가 전에 query log 작업하면서 넣어둔 건데 이제 필요 없어서 삭제해주심 감사하겠습니다.